### PR TITLE
Update intro wording, OAS definition [file] fixup

### PR DESCRIPTION
**Pull request by MikeRalphson**
* Borrows wording from https://www.openapis.org/specification/repo
* Changes definition of **OpenAPI Definition File** ("a file or set of files") to **OpenAPI Definition**

### CODE REVIEW

File: versions/3.0.md [hunk 1]
```diff
@@ -70,8 +66,8 @@ Additional utilities, such as testing tools, can also use the files.
 
 ## Definitions
 
-##### <a name="oasDefinitionFile"></a>OpenAPI Definition File
-A file or set of files which defines an API. The OpenAPI definition file uses and conforms to the OpenAPI Specification.
+##### <a name="oasDefinition"></a>OpenAPI Definition
+A file or set of files which defines an API. An OpenAPI definition uses and conforms to the OpenAPI Specification.
```
Comment (ePaul) [inline versions/3.0.md]: Hmm, maybe use here "document or set of documents", as these things don't need to be in a file system at all?

(Similarly, I would replace `file` with `document everywhere where it is still left now.)

**Comment by MikeRalphson** [comment]
Updated so references to files -> documents except where relating to file uploads.

**Comment by earth2marsh** [comment]
Changes LGTM, but they got me thinking about lines 13 and 70 and how "description" may be more appropriate than "definition", since description is a superset of definition (possible truth vs truth). I'll make a separate PR after discussion once this PR is merged.

**Comment by RobDolinMS** [comment]
#TDC: @earth2marsh should do a separate PR for description vs. definition

**Pull request merged**